### PR TITLE
AB#104413 Rename DocumentService to UtilsService

### PR DESCRIPTION
**Pull request by Carq**

### CODE REVIEW

**Review by dawid-pawulski — APPROVED** [review]

File: src/HE.UtilsService/.idea/.idea.HE.DocumentService/.idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jjankowski87) [inline src/HE.UtilsService/.idea/.idea.HE.DocumentService/.idea/workspace.xml:1]: `.idea` directory should be ignored

File: src/HE.UtilsService/HE.UtilsService.Api/HE.UtilsService.Api.csproj [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <RootNamespace>HE.DocumentService.Api</RootNamespace>
```
Comment (jjankowski87) [inline src/HE.UtilsService/HE.UtilsService.Api/HE.UtilsService.Api.csproj:7]: hmm, why we want to keep this old namespace?

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.UtilsService/.idea/.idea.HE.DocumentService/.idea/workspace.xml (hunk 1 above)
Comment (Carq) [reply src/HE.UtilsService/.idea/.idea.HE.DocumentService/.idea/workspace.xml]: Fixed. Thanks for spotting it.

File: src/HE.UtilsService/HE.UtilsService.Api/HE.UtilsService.Api.csproj (hunk 2 above)
Comment (Carq) [reply src/HE.UtilsService/HE.UtilsService.Api/HE.UtilsService.Api.csproj]: My bad, fixed. Thanks JJ.

**Pull request merged**